r/R/compute.R

```diff
@@ -95,10 +95,14 @@ list_compute_functions <- function(pattern = NULL, ...) {
 }
 
 #' @export
-sum.ArrowDatum <- function(..., na.rm = FALSE) scalar_aggregate("sum", ..., na.rm = na.rm)
+sum.ArrowDatum <- function(..., na.rm = FALSE, na.min_count = 0) {
+  scalar_aggregate("sum", ..., na.rm = na.rm, na.min_count = na.min_count)
+}
```

Comment:
`na.min_count` will propagate through `...` if anyone passes it (but they won't).

```suggestion
sum.ArrowDatum <- function(..., na.rm = FALSE) {
  scalar_aggregate("sum", ..., na.rm = na.rm)
}
```